### PR TITLE
update inheritance diagrams

### LINKED ISSUE #721: API ref: Improve inheritance diagram visuals
#670 introduced inheritance diagrams in our API reference but there can be many improvements made.

- [Graphviz parameters](https://www.graphviz.org/doc/info/attrs.html) to improve visuals 
- CSS for formatting
- `.rst` templates for API reference

### PR DESCRIPTION
**Pull request by bchen1116**
fix #721 

Docs [here](https://evalml.alteryx.com/en/bc_721_visual/generated/evalml.pipelines.MulticlassClassificationPipeline.html#evalml.pipelines.MulticlassClassificationPipeline)

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -214,8 +214,9 @@
 nbsphinx_execute = 'always'
 nbsphinx_timeout = 600 # sphinx defaults each cell to 30 seconds so we need to override here
 
-inheritance_graph_attrs = dict(rankdir="LR", size='"1000, 333"',
-                               fontsize=30, labelfontsize=30, ratio='compress', dpi=960)
+inheritance_graph_attrs = dict(rankdir="TB", dpi=192.0, ranksep=0.02)
+inheritance_node_attrs = dict(fontsize=6, height=0.15, width=5, fixedsize=True)
```
Comment (bchen1116) [inline docs/source/conf.py:218]: I wasn't able to figure out how to left-align the text. `label=r'\l'` created errors in rendering the docs.

File: docs/source/_static/style.css [hunk 2]
```diff
@@ -1,11 +1,4 @@
 div.graphviz {
     overflow: auto;
-    max-width: 620px;
-    max-height: 50px;
-    text-align: center;
+    text-align: left;
```
Comment (dsherry) [inline docs/source/_static/style.css:3]: Weird, so `graphviz` isn't respecting this.

File: docs/source/_templates/class.rst [hunk 3]
```diff
@@ -18,3 +16,8 @@
    {%- endfor %}
    {% endif %}
    {% endblock %}
+
+Class Inheritance
```
Comment (dsherry) [inline docs/source/_templates/class.rst:20]: I like it!

File: docs/source/conf.py (hunk 1 above)
Comment (dsherry) [reply docs/source/conf.py:218]: No worries, this is still great :)

**Review by dsherry — APPROVED** [review]
@bchen1116 thanks for wrestling with this! I think the new output is easier to look at and we should 🚢  :)

File: docs/source/_static/style.css (hunk 2 above)
Comment (bchen1116) [reply docs/source/_static/style.css:3]: @dsherry Yea, when I look at the html and css styling of the rendered page, this `graphviz` element is a larger container that contains the rendered image, so any text-aligning done here won't have an impact on the image contents.

**Pull request merged**